net.lr.example.kubernetes/src/main/java/net/lr/ds/hello/ReadyCheck.java

```diff
@@ -45,7 +48,7 @@ public void notReadyForSeconds(int seconds) {
 	}
 	
 	@Override
-	protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+	protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
```

Comment:
Technically post makes sense as we do some action here. The advantage of get is that you can type it in the browser. I do not have a strong opinion about this though. So if you prefer it that is fine.

Reply:
I kind of do. There's a reason why it's wrong and I've personally found myself refreshing the wrong page, with this example. So I guess we can keep it.

Reply:
+1